keyboards/ergodox_ez/util/keymap_beautifier.py

```diff
@@ -0,0 +1,154 @@
+                if line == "const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {":
+                    current_section = "keys"
+                    src[current_section].append("const int keymaps[]={")
+                else:
+                    src[current_section].append(line)
+            elif current_section == "keys":
+                src[current_section].append(line)
+                if line == "};":
+                    current_section = "after"
+            else:
+                src[current_section].append(line)
+
+    output = "\n".join(src["before"]) + "\n" 
+    output += beautify_keys_section("\n".join(src["keys"])) + "\n"
+    output += "\n".join(src["after"]) + "\n"
+    return output
+
+def beautify_keys_section(src):
+    #parsed = pycparser.parse_file(c_filename)
+    parsed = pycparser.CParser().parse(src)
+    layer_output = []
+    for layer in parsed.children()[0][1].init.exprs:
+        ascii_key_symbols = transform_layer(layer)
+        layer_output.append(pretty_output_layer(layer.name[0].value, ascii_key_symbols))
+    return """
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {{
+/*
+ * ,--------------------------------------------------.    ,--------------------------------------------------.
+ * |    0   |   1  |   2  |   3  |   4  |   5  |  6   |    |  38  |  39  |  40  |  41  |  42  |  43  |   44   |
+ * |--------+------+------+------+------+------+------|    |------+------+------+------+------+------+--------|
+ * |    7   |   8  |   9  |  10  |  11  |  12  |  13  |    |  45  |  46  |  47  |  48  |  49  |  50  |   51   |
+ * |--------+------+------+------+------+------|      |    |      |------+------+------+------+------+--------|
+ * |   14   |  15  |  16  |  17  |  18  |  19  |------|    |------|  52  |  53  |  54  |  55  |  56  |   57   |
+ * |--------+------+------+------+------+------|  26  |    |  58  |------+------+------+------+------+--------|
+ * |   20   |  21  |  22  |  23  |  24  |  25  |      |    |      |  59  |  60  |  61  |  62  |  63  |   64   |
+ * `--------+------+------+------+------+-------------'    `-------------+------+------+------+------+--------'
+ *   |  27  |  28  |  29  |  30  |  31  |                                |  65  |  66  |  67  |  68  |  69  |
+ *   `----------------------------------'                                `----------------------------------'
+ *                                       ,-------------.  ,-------------.
+ *                                       |  32  |  33  |  |  70  |  71  |
+ *                                ,------+------+------|  |------+------+------.
+ *                                |      |      |  34  |  |  72  |      |      |
+ *                                |  35  |  36  |------|  |------|  74  |  75  |
+ *                                |      |      |  37  |  |  73  |      |      |
+ *                                `--------------------'  `--------------------'
+ */
+{}
```

Comment:
I recommend using an f-string. Change line 41 to `return f"""`, then change this line to:
```suggestion
{'\n'.join(layer_output)}
```
and remove the `format` call on line 66.